backend/src/services/course/course.routes.js

```diff
@@ -72,6 +71,7 @@ module.exports = async function courseRoutes(fastify, options) {
   fastify.get(
     '/courses/:course_id/users',
     {
+      preHandler: [fastify.requireTAOrProfessorInCourse],
```

Comment:
[Blocker]
Enrolled students should also be able to see a list of other students in the class.